src/main/java/com/infusion/relnotesgen/Configuration.java

```diff
@@ -50,12 +50,15 @@
     static final String ISSUE_SORT_PRIORITY = "issue.sort.priority";
     static final String REPORT_DIRECTORY = "report.directory";
     static final String REPORT_TEMPLATE = "report.template";

```

Comment:
what's the difference between `REPORT_TEMPLATE` and `REPORT_EXTERNAL_TEMPLATE`?
Is the first one still needed?